client/my-sites/plans/current-plan/jetpack-product-install/index.tsx

```diff
@@ -233,20 +255,44 @@ export class JetpackProductInstall extends Component {
 	}
 }
 
+function mapStateToProps( state ) {
+	const siteId = getSelectedSiteId( state );
+	const queryArgs = getCurrentQueryArguments( state );
+
+	const installQuery: string[] =
+		// eslint-disable-next-line no-nested-ternary
+		typeof queryArgs === 'object' && 'install' in queryArgs
+			? Array.isArray( queryArgs.install )
```

Comment:
You could use `_.castArray(queryArgs.install)` to collapse a branch of the "beautiful nested ternary" ;)